content/developer/reference/data.rst

```diff
@@ -276,7 +276,7 @@ For this case, data files can also use csv_, this is often the case for
   for :term:`external identifiers` (used for creation or update)
 * each row thereafter creates a new record
 
-Here's the first lines of the data file defining US states
+Here's the first lines of the data file defining Australian and US states
```

Comment:
```suggestion
Here's the first lines of the data file defining states
```
while that's not visible on the screenshot (because it's sorted by country code) the file contains way the hell more than just US and AU states (most of which I don't think are really used day-to-day).